plugins/rocks.xml.webhelp/xsl/topic.xsl

```diff
@@ -212,10 +208,24 @@
     </xsl:template>
 
     <xsl:template match="*" mode="addFooterToHtmlBodyElement">
+        <xsl:variable name="organization-name">
+            <xsl:choose>
+                <xsl:when
+                        test="$input.map/descendant::*[contains(@class, ' bookmap/bookowner ')]/child::*[contains(@class, ' bookmap/organization ')]">
+                    <xsl:value-of
+                            select="$input.map/descendant::*[contains(@class, ' bookmap/bookowner ')]/child::*[contains(@class, ' bookmap/organization ')][1]"/>
+                </xsl:when>
+                <xsl:otherwise>
+                    <xsl:text>Organization name ©</xsl:text>
```

Comment:
I don't think we should hardcode 'Organisation name' string here. It would be better to remove this text at all. We should add an ability to pass the organisation name from a cmd if user needs this data but don't have it in the input map. Please add one more parameter for this.